pkgs/applications/networking/instant-messengers/element/element-desktop.nix

```diff
@@ -56,6 +58,10 @@ mkYarnPackage rec {
     runHook postBuild
   '';
 
+  darwinIconCmds = ''
+    makeicns -in $out/share/element/electron/build/icons/512x512.png -out $out/share/element/electron/build/icons/element.icns
```

Comment:
Any reason to not use `${libicns}/bin/png2icns`?

Reply:
Nice find! I didn't even know that `libicns` existed and was already packaged.  I suppose that makes `makeicns` redundant in that case?

Reply:
If `libicns` works for your case, it could be the better option since it's a bit more modern and is written in C, so it would be a bit easier to maintain.

Reply:
One big disatvantage of makeicns is that it doesn't work from inside of our darwin sandbox.